yoyodyne/indexes.py

```diff
@@ -13,6 +13,7 @@ class SymbolMap:
     symbol2index: Dict[str, int]
 
     def __init__(self, vocabulary: List[str]):
+        # Keep special.SPECIAL first to maintain overlap with features.
```

Comment:
Minor style thing but should be "Keeps" since it's like, telegraphic declarative sentences.